backend/uclapi/oauth/views.py

```diff
@@ -763,6 +763,10 @@ def deauthorise_app(request):
 
 @ensure_csrf_cookie
 def logout(request):
```

Comment:
I've been thinking about the point of logout, especially considering issue #970. The conclusion from the issue, AFAICT is that we shouldn't do logout, as it is misleading from a security PoV.

Considering the security aspect, I think we should remove the logout button, view, and url pattern completely. People will logout, and then may give their laptop to someone else, assuming they can't login, when in fact they can. If we remove the option to logout, then giving your laptop to someone else is a security flaw committed by the user willingly, knowing the dangers.

Reply:
Library has the same issue. Their solution is to ask people to close their browser window after they've logged out. This works presumably because shibb cookies are session cookies and expire once the browser session closes.

Perhaps we can keep the logout button but include a warning explaining that users aren't really logged out until they close their browser: e.g.

>You have been logged out of UCL API. However, you are still logged in to other UCL services. To log out completely, please close your web browser.

Reply:
I am happy to do either, I feel like i we add a warning explaining that you need to close your browser it at least gives the user the option to be able to log out and instructions on the website without clearing all of their cookies.

I can add this in later, do you think the warning that they need to close the browser to completely log out is sufficient @Feverfew